Gemfile.lock

```diff
@@ -134,18 +134,6 @@ PATH
     veteran_confirmation (0.0.1)
     vye (0.1.0)
 
-GEM
```

Comment:
Please put this back. I'm assuming your local dev env removed this on accident

Reply:
Yeah, this wasn't meant to get pushed, will fix